bot/src/main.rs

```diff
@@ -1654,6 +1663,10 @@ fn classify(
             })
             .collect();
 
+        if config.require_performance_metrics_reporting && poor_reporters.is_empty() {
```

Comment:
wouldn't `poor_reporters.is_empty()` not be empty in epochs we want to discard--because they would be filled with poor (non-metric reporting) reporters)?

Reply:
`poor_reporters` contains validators who did not report in 8/10 of the last epochs. And epochs where everyone reported poorly are not included in the 10 (see https://github.com/solana-foundation/stake-o-matic/blob/acc181ed4348df97c84794ef94007023e0869b06/bot/src/main.rs#L1568-L1574).

I'll rename `poor_reporters` to `poor_reporters_last_10_epochs` to hopefully make this more clear.